modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/registry/ObjectServiceUpgradeStepRegistrator.java

```diff
@@ -516,7 +517,10 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 				"ObjectDefinition", "enableFriendlyURLCustomization BOOLEAN"));
 
 		registry.register(
-			"10.3.0", "10.4.0", ObjectEntryFolderTable.create(),
+			"10.3.0", "10.3.1", new ObjectAssetTitleUpgradeProcess());
+
+		registry.register(
+			"10.3.1", "10.4.0", ObjectEntryFolderTable.create(),
```

Comment:
@yuriymr Shouldn't you create a new upgrade process instead of changing an existing one?

@guilhermedcamacho

Reply:
@yuriymr you should always add a new upgrade step to the end of this class.  Otherwise, existing users at schema version 10.8.0 (for example) will never execute your upgrade.

Reply:
Thanks @carolmariaabb